cmd/bucket-handlers.go

```diff
@@ -292,6 +294,16 @@ func (api objectAPIHandlers) ListBucketsHandler(w http.ResponseWriter, r *http.R
 			writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL, guessIsBrowserReq(r))
 			return
 		}
+
+		for i := range bucketsInfo {
+			configFile := path.Join(bucketConfigPrefix, bucketsInfo[i].Name, bucketCreatedTimeFile)
```

Comment:
This is a significant overhead, just by the IO alone. Do we have to read through this all the time?

Reply:
This is not expected to be called frequently like say list objects. This is supposed to be a not so frequent call.